Fix wrong benchmark

mikhailramalho opened the pull request:
Based on the fixes from commit bcc887.

Although this doesn't lead to UB, the assertion will eventually fail when sn wrap around.

Signed-off-by: Mikhail Ramalho <mikhail.ramalho@gmail.com>

PhilippWendler commented:
If this counterexample is confirmed, I would suggest to keep the original version of the task labelled as `false` in addition to the corrected one.

mdangl commented:
This is fixed (and the bug preserved in a separate task with changed label) in #202.
Sorry about that: had I seen this pull request, I would not have made a new one.

The fix in #202, however, is closer to the original task, because it preserves the large loop bound.

mikhailramalho commented:
Updated. Added the original wrong version.

2016-11-09 16:55 GMT+00:00 Philipp Wendler <notifications@github.com>:

> If this counterexample is confirmed, I would suggest to keep the original
> version of the task labelled as false in addition to the corrected one.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sosy-lab/sv-benchmarks/pull/200#issuecomment-259464953>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABREn5UOnobE6lP9SXL4y-kRFkk3H7e_ks5q8fr1gaJpZM4KttA9>
> .
>



-- 

Mikhail Ramalho.

kfriedberger commented:
Already fixed in repository, see #202. Closed.
